src/SFA.DAS.EmployerIncentives.Api/Startup.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using SFA.DAS.EmployerIncentives.Api.AppStart;
+using SFA.DAS.EmployerIncentives.Api.Authentication;
+using SFA.DAS.EmployerIncentives.Api.Authorization;
+using SFA.DAS.EmployerIncentives.Api.Configuration;
+using SFA.DAS.EmployerIncentives.Api.ErrorHandler;
+using SFA.DAS.EmployerIncentives.Api.HealthChecks;
+using IHostingEnvironment = Microsoft.AspNetCore.Hosting.IHostingEnvironment;
+
+namespace SFA.DAS.EmployerIncentives.Api
+{
+    public class Startup
+    {
+        private readonly IWebHostEnvironment _env;
+        private readonly IConfiguration _configuration;
+
+        public Startup(IConfiguration configuration, IWebHostEnvironment env)
+        { _env = env;
+            _configuration = configuration;
```

Comment:
This should be added to the ConfigurationBuilder

Reply:
done